README.md

```diff
@@ -49,8 +49,8 @@ terraform {
 
 # Credentials can be found at www.twilio.com/console.
 provider "twilio" {
-  //  account_sid defaults to TWILIO_ACCOUNT_SID env var
-  //  auth_token  defaults to TWILIO_AUTH_TOKEN env var
+  //  account_sid defaults to TWILIO_ACCOUNT_SID/TWILIO_API_KEY env var
```

Comment:
Can you add some documentation around the precedence order for credentials. What takes precedence if a customer has an `TWILIO_ACCOUNT_SID` and `TWILIO_API_KEY` defined.